webpack.config.js

```diff
@@ -75,7 +75,9 @@ const webpackConfig = {
                     loader: 'babel-loader',
                     options: {
                         babelrc: false,
-                        presets: ['@babel/preset-env'],
+                        presets: [
+                            ['@babel/preset-env', { loose: true, modules: false }]
```

Comment:
@mamaddox made a similar change i think with the typescript that caused some issues but i don't remember if it was these exact options.

Reply:
yeah i think the `class check` specifically caused issues with the way we bundle related but i'll let maria confirm

Reply:
These were the options I removed. But, there were some tsconfig settings that were definitely causing some issues. I think the config settings were the root of the cause tbh, but I need to verify first.

Reply:
`modules: false` is probably safe. It's `loose` mode that really saves space by cutting out a boiler plate code.

The test failure we're seeing in commercial (not this project) is that the `get displayMode` override in `PlaylistController`  overrides the getter in `RecsController` but falls back to calling the getter on the base class with `super.displayMode` < that's where it breaks. Updating webpack and babel doesn't resolve it, but the fix is "super" simple:

https://github.com/jwplayer/jwplayer-commercial/pull/7395/commits/fd1734f268e99be3802bf400747d99dd49062877

I tried but could not reproduce the issue in Babel repl or find any open issues.
https://babeljs.io/en/repl#?browsers=&build=&builtIns=false&spec=false&loose=true&code_lz=MYGwhgzhAEBCkFNoG8BQ1oHMEBdo4QhwAoBKFdDaAJ1wFdqA7aARgG5KBfVb1USGAHkARgCtoCAB4FGAExjwISNBgD0q6IIBuCatQCWspNhwFq0MHOh0l0CHQAOuyifyES5FVX0AzaMR8wECVyWhwGZgAmDioaeiY7R10AOgIiGOhuXmAAe0YiaByxaABeaEYEAHdNMTIOXPyckARkkBzMYiLRVPdSDiA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=true&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2%2Cenv&prettier=false&targets=&version=7.8.3&externalPlugins=

Reply:
`modules: false` isn't probably needed, since it's already the default behavior when using Webpack.